store/def_query_tree_index.go

```diff
@@ -43,6 +43,11 @@ func (x *defQueryTreeIndex) getByQuery(q string) (map[unit.ID2]byteOffsets, bool
 		panic("mafsaTable not built/read")
 	}
 
+	if x.mt.t == nil {
+		log.Println("getByQuery: x.mt.t == nil")
```

Comment:
Hmm, this should probably be `vlog` to be consistent with the logging in this package?